x-pack/plugins/security/server/config.ts

```diff
@@ -200,6 +200,18 @@ export const ConfigSchema = schema.object({
   }),
   audit: schema.object({
     enabled: schema.boolean({ defaultValue: false }),
+    appender: schema.maybe(coreConfig.logging.appenders),
+    ignore_filters: schema.maybe(
+      schema.arrayOf(
+        schema.object({
+          actions: schema.maybe(schema.arrayOf(schema.string())),
+          categories: schema.maybe(schema.arrayOf(schema.string())),
+          types: schema.maybe(schema.arrayOf(schema.string())),
+          outcomes: schema.maybe(schema.arrayOf(schema.string())),
+          spaces: schema.maybe(schema.arrayOf(schema.string())),
+        })
+      )
+    ),
```

Comment:
Since `ignore_filters` is only meant to be used with the new audit logger, we should probably prevent them from being specified if `appender` isn't also configured. This will hopefully make it clearer that you can't filter out events from the legacy logger. Something like the following should work:

```suggestion
    ignore_filters: schema.conditional(
      schema.siblingRef('appender'),
      coreConfig.logging.appenders,
      schema.maybe(
        schema.arrayOf(
          schema.object({
            actions: schema.maybe(schema.arrayOf(schema.string())),
            categories: schema.maybe(schema.arrayOf(schema.string())),
            types: schema.maybe(schema.arrayOf(schema.string())),
            outcomes: schema.maybe(schema.arrayOf(schema.string())),
            spaces: schema.maybe(schema.arrayOf(schema.string())),
          })
        )
      ),
      schema.never()
    ),
```